### PR TITLE
Update links to "voice-change-o-matic- repository

### PR DESCRIPTION
**Pull request by bsmth**
__Context:__

The https://github.com/mdn/voice-change-o-matic repo has been migrated into `mdn/webaudio-examples` in https://github.com/mdn/webaudio-examples/pull/92.

__Changes in this PR:__
This PR updates all links to the old repository to point instead to https://github.com/mdn/webaudio-examples/tree/main/voice-change-o-matic.
Also updated the links to code examples with line numbers.

### CODE REVIEW

File: files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md [hunk 1]
```diff
@@ -105,9 +105,9 @@ Let's start by creating our own canvas that we draw future experiments on to.
 3. Now open "script.js" and add the following lines of JavaScript:
 
    ```js
-   const canvas = document.querySelector('.myCanvas');
-   const width = canvas.width = window.innerWidth;
-   const height = canvas.height = window.innerHeight;
+   const canvas = document.querySelector(".myCanvas");
+   const width = (canvas.width = window.innerWidth);
+   const height = (canvas.height = window.innerHeight);
```
Comment (teoli2003) [inline files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md:110]: Has this been added by Prettier?

File: files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md [hunk 2]
```diff
@@ -638,22 +634,22 @@ let curY;
...
 
 // update mouse pointer coordinates
-document.addEventListener('mousemove', (e) => {
+document.addEventListener("mousemove", (e) => {
   curX = e.pageX;
   curY = e.pageY;
 });
 
-canvas.addEventListener('mousedown', () => pressed = true);
+canvas.addEventListener("mousedown", () => (pressed = true));
```
Comment (teoli2003) [inline files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md:642]: Same here.

File: files/en-us/web/api/web_audio_api/visualizations_with_web_audio_api/index.md [hunk 3]
```diff
@@ -89,52 +89,52 @@ function draw() {
...
+canvasCtx.strokeStyle = "rgb(0, 0, 0)";
+canvasCtx.beginPath();
 ```
 
 Determine the width of each segment of the line to be drawn by dividing the canvas width by the array length (equal to the FrequencyBinCount, as defined earlier on), then define an x variable to define the position to move to for drawing each segment of the line.
 
 ```js
-  const sliceWidth = WIDTH * 1.0 / bufferLength;
-  let x = 0;
+const sliceWidth = (WIDTH * 1.0) / bufferLength;
```
Comment (teoli2003) [inline files/en-us/web/api/web_audio_api/visualizations_with_web_audio_api/index.md:119]: Another weirdness of Prettier, while 6 y.o. children are learning mathematical * and / precedence in school.

By the way, we could just write `WIDTH / bufferLength`

**Review by teoli2003 — APPROVED** [review]
A few Prettier weirdness, but I think this is ready to go.

File: files/en-us/web/api/web_audio_api/visualizations_with_web_audio_api/index.md (hunk 3 above)
Comment (bsmth) [reply files/en-us/web/api/web_audio_api/visualizations_with_web_audio_api/index.md:119]: >By the way, we could just write WIDTH / bufferLength

Good point... Let me fix this

File: files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md:110]: It has been added by Prettier, yes. It looks like it's automatically adding brackets for

```
a = b = c
```

Any preferences on how this should look? Is this incorrect to add parens?

Comment (teoli2003) [reply files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md:110]: No, it is fine (not my preference, but that's not important)

Comment (bsmth) [reply files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md]: I've reverted in 88c6ce60d8f1927d36cd19d6c5043df87f18c485 anyway. Let's go with this for now 👍🏻

File: files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md (hunk 2 above)
Comment (bsmth) [reply files/en-us/learn/javascript/client-side_web_apis/drawing_graphics/index.md]: Done in 88c6ce60d8f1927d36cd19d6c5043df87f18c485

File: files/en-us/web/api/web_audio_api/visualizations_with_web_audio_api/index.md (hunk 3 above)
Comment (bsmth) [reply files/en-us/web/api/web_audio_api/visualizations_with_web_audio_api/index.md]: Done in 88c6ce60d8f1927d36cd19d6c5043df87f18c485

**Pull request merged**